docs/src/examples/components/Form/Types/FormExample.shorthand.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Form, Input, Button } from '@stardust-ui/react'
+
+const fields = [
+  {
+    label: 'First name',
+    controlType: Input,
+    name: 'firstName',
+    id: 'first-name-shorthand',
+    key: 'first-name',
+    required: true,
+  },
+  {
+    label: 'Last name',
+    controlType: Input,
+    name: 'lastName',
+    id: 'last-name-shorthand',
+    key: 'last-name',
+    required: true,
+  },
+  {
+    label: 'I agree to the Terms and Conditions',
+    controlType: 'input',
+    type: 'checkbox',
+    id: 'conditions-shorthand',
+    key: 'conditions',
+  },
+  { controlType: Button, control: { content: 'Submit' }, key: 'submit' },
+]
+
+const FormExample = () => (
+  <Form
+    onSubmit={() => {
+      alert('Form submitted')
+    }}
+    fields={fields}
```

Comment:
nit: when this example is rendered, it seems that there is no any gap provided between checkbox and text field:

![image](https://user-images.githubusercontent.com/9024564/47159674-acec0b00-d2ee-11e8-919a-5d6c05346e94.png)

Could we address this problem in styles?

Reply:
Fixed for the checkbox and radio type inputs

Reply:
Marija, this the rendering result that I had when tried the examples: 

![image](https://user-images.githubusercontent.com/9024564/47183515-86e35c80-d328-11e8-831c-5ef3ca412797.png)

Please, ping me if you'll need help to address that - probably this is result of recent changes made related to `controlType`, will be more than happy to help

Reply:
This is regression from the latest changes of the Input component. After merging the latest master, using
```jsx
<Slot as={Input} />
```
renders the html:
```html
<div class="ui-slot ui-input ui-slot">div</div>
```
It works as expected with the RadioGroup, input, Button etc. as well as with the previous implementation of the Input component. Will need to investigate this with Alex tomorrow.